source/_integrations/cover.rflink.markdown

```diff
@@ -168,6 +168,38 @@ cover:
 
 The configuration above shows that the `type` property may be omitted. When the ID starts with `newkaku`, the component will make sure that the on and off commands are inverted. When the ID does not start with `newkaku`, the on and off commands are not inverted.
 
+## Setting up a non-RTS cover
+
+Configure `automatic_add` for the light domain (yes, the light domain)
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: rflink
+    automatic_add: true
+```
+
+When you press the remote buttons, a new light will be available at the `developer-tools → states`.
```

Comment:
Should we link to the entities dashboard instead (using my link):

[![Open your Home Assistant instance and show your entities.](https://my.home-assistant.io/badges/entities.svg)](https://my.home-assistant.io/redirect/entities/)

Reply:
Agree.  Committed change.